debian/compat

```diff
@@ -0,0 +1 @@
+9
```

Comment:
It is not required.
But we will get warning without it:

_No compatibility level specified in debian/compat
This package will soon FTBFS; time to fix it!_